### DISCUSSION TITLE
Browsing by Volume should show Issues in that volume - not open it to read from the start.

### DISCUSSION THREAD

**Discussion by maximo1984** [Kareadita/Kavita#2752] [Ideas]
### Idea Description

Comic Volumes can grow very large, with 100's of issues. Combine an entire history of a Comic series, by volumes and we are talking about 1000's of issues. Just like browsing a Season of a Tv show on someting like Jellyfin, I would expect clicking a volume of a comic series to show only those issues in that volume.

### Idea Category

API

### Duration of Using Kavita

1 week.

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by moisesbelda** [comment]
It is possible to go to a volume specific issues to open one of them through:
- Go to the Volume card, click on the 3 buttons icon and select Details.
- A modal windows will be displayed on the bottom, select Info section.
- A list of the issues related to this volume will be presented. If you click on one of them, it will be opened directly to the comic browser.
I upvoted this feature request in order to get this list of specific issues of a volume more accesible.

**Comment by majora2007** [comment]
I believe we can consider this closed now. Kavita now supports the ability to open individual volumes and view their issues in the same "series-detail" screen as if it were a series.
